packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/audit/requests/ExportRequest.java

```diff
@@ -0,0 +1,128 @@
+package cloud.pangeacyber.pangea.audit.requests;
+
+import cloud.pangeacyber.pangea.BaseRequest;
+import cloud.pangeacyber.pangea.audit.models.DownloadFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Instant;
+
+@JsonInclude(Include.NON_NULL)
+public final class ExportRequest extends BaseRequest {
+
+	/** Format for the records. */
+	@JsonProperty("format")
+	private final DownloadFormat format;
+
+	/** The start of the time range to perform the search on. */
+	@JsonProperty("start")
+	private final Instant start;
+
+	/**
+	 * The end of the time range to perform the search on. If omitted, then all
+	 * records up to the latest will be searched.
+	 */
+	@JsonProperty("end")
+	private final Instant end;
```

Comment:
Hmm, same comment than in go, do we want a timestamp here instead a date or a string?

Reply:
Updated them to strings.